tests/online_migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -31,7 +31,51 @@ sub run {
         # set blank screen to be never for current session
         script_run("gsettings set org.gnome.desktop.session idle-delay 0");
         become_root;
+        if (check_var('HDDVERSION', '12')) {
+            # use latest yast2-registration version because is not officially available
+            assert_script_run 'wget --quiet ' . data_url('yast2-registration-3.1.129.18-1.noarch.rpm');
+            zypper_call '--no-gpg-checks in yast2-registration-3.1.129.18-1.noarch.rpm';
+            record_info 'workaround', 'until the package is available for SLE 12 GA';
```

Comment:
@dzedro This change causes the issue: https://progress.opensuse.org/issues/19522

Also I can see the version of yast2-registration is shown as following image if fully patch applied before migration on SLE12 GA:
![screenshot_20170606_122259](https://cloud.githubusercontent.com/assets/9360645/26814943/bb7b85ae-4abb-11e7-8bf5-f41d6ba0259b.png)
Does it meet the requirement to test new behaviours? 
If so, `if (check_var('HDDVERSION', '12') && !get_var(FULL_UPDATE))` is better.